src/processor.js

```diff
@@ -930,6 +929,9 @@ module.exports = class Processor {
         if (this.options.skipWriteBlockInfoRoot === true) {
             this.starkInput.skipWriteBlockInfoRoot = true;
         }
+
+        // add DB
+        this.starkInput.db = await getCurrentDB(this.oldStateRoot, this.db, this.F);
```

Comment:
Why this? There is any difference or is a matter of cleaner code?

Reply:
When the object `starkInput` is saved in a JSON file, it saves all its properties in the same order as they are inserted in the object.
Then the `db` object will be the last one always.
I would rather see readable properties at the beginning and the then the DB as the last property